Amplify/Categories/Auth/Result/AuthSignInResult.swift

```diff
@@ -14,14 +14,14 @@ public struct AuthSignInResult {
     /// When this value is false, it means that there are more steps to follow for the signIn flow. Check `nextStep`
     /// to understand the next flow. If `isSignedIn` is true, signIn flow has been completed.
     public var isSignedIn: Bool {
-        nextStep.signInStep == .done
+        nextStep == .done
```

Comment:
I'd prefer not to add `Equatable` conformance just for this, if this is the only use case.

Reply:
Removed Equatable